ProjectTraning/Program.cs

```diff
@@ -10,21 +10,415 @@ class Program
+
+        }
+
+        public static void Task13()
+        {
+            Console.WriteLine("Add Number");
+            int number = Convert.ToInt32(Console.ReadLine());
+            int renumber = 0;
+            while (number > 0)
+            {
+                renumber *= 10;
+                renumber += number % 10;
+                number /= 10;
+            }
+            Console.WriteLine($"Overturn:{renumber}");
+        }
+
+        public static void Task2()
+        {
+            int a, b, c;
+            for (a = 1; a <= 5; a++)
+            {
+                for (b = 4; b >= a; b--)
+                {
+                    Console.Write(" ");
+                }
+                for (c = 1; c <= (2 * a - 1); c++)
+                {
+                    Console.Write("*");
+                }
+                Console.WriteLine();
+            }
+            Console.ReadLine();
+        }
 
-        public static decimal PerformCalculation(int r)
+        public static void Task14()
         {
-            const decimal Pi = 3.14m;
+            Console.WriteLine("Add Number");
+            string number = Console.ReadLine();
+            int number2 = Convert.ToInt32(number);
+            int renumber = 0;
+            int temp;
+            while (number2 > 0)
+            {
+                renumber *= 10;
+                renumber += number2 % 10;
+                number2 /= 10;
+            }
+            Console.WriteLine("Numbers:");
+            for (int i = 0; i < number.Length; i++)
+            {
+                temp = renumber % 10;
+                renumber /= 10;
 
-            decimal square = Pi * r * r;
+                Console.WriteLine(temp);
+            }
 
-            return square;
         }
+
+        public static void Task3()
+        {
+            Console.WriteLine("Add Number");
+            int number = Convert.ToInt32(Console.ReadLine());
+            bool b = true;
+            string text = "";
+            if (number >= 0)
+            {
+                int temp = number;
+                for (int i = 2; i <= number; i++)
+                {
+                    b = true;
+                    for (int j = 2; j < i; j++)
+                    {
+                        if (i % j == 0)
+                        {
+                            b = false;
+                            break;
+                        }
+                    }
+                }
+                if (b == true)
+                {
+                    Console.WriteLine("Prime number");
+                }
+                else
+                {
+                    Console.WriteLine("Not prime number");
+                }
+                Console.WriteLine("Positive Number");
+
+                if (temp % 2 == 0)
+                {                    
+                    text += "2" + " ";
+                }
+                if (number % 5 == 0)
+                {                   
+                    text += "5" + " ";
```

Comment:
Also no need to use string concatenation since you don't use variables. Just add space to existing literal.